crates/ui/src/root.rs

```diff
@@ -57,7 +57,14 @@ impl<'a> ContextModal for WindowContext<'a> {
             if root.active_drawer.is_none() {
                 root.previous_focus_handle = cx.focused();
             }
-            root.active_drawer = Some(Rc::new(build));
+
+            let focus_handle = cx.focus_handle();
+            focus_handle.focus(cx);
```

Comment:
We could't do `focus` here, this will make some unexpected bug.

Reply:
I was found this in Modal before, it's was also used `focus`, then I found some issue, so it now removed.

Reply:
The notification layout not display now.